spring-boot-web-application/src/main/java/com/in28minutes/springboot/service/StudentService.java

```diff
@@ -39,4 +42,4 @@ public List<Course> retrieveCourses(String studentId) {
         return student == null ? null : student.courses();
 
     }
-}
\ No newline at end of file
+}
```

Comment:
1. Missing `students.add(slava);` to add the new student to the list.
2. No newline at the end of the file.